src/blocked.c

```diff
@@ -97,6 +99,22 @@ void blockClient(client *c, int btype) {
     }
 }
 
+/* This function is called after a client has finished a blocking operation
+ * in order to update the total command duration, log the command into
+ * the Slow log if needed, and log the reply duration event if needed. */
+void updateStatsOnUnblock(client *c, ustime_t blocked_us, ustime_t reply_us){
+    const ustime_t total_cmd_duration = c->duration + blocked_us + reply_us;
+    c->lastcmd->microseconds += total_cmd_duration;
+    /* Log the command into the Slow log if needed. */
+    if (!(c->lastcmd->flags & CMD_SKIP_SLOWLOG)) {
+        const char *latency_event = (c->lastcmd->flags & CMD_FAST) ?
+                                    "fast-command" : "command";
```

Comment:
i think i would change the latency event be "command-unblocking".
reasoning here: https://github.com/redis/redis/pull/7491#issuecomment-657201892

Reply:
addressed in the latest commit.